src/blocks/_fhi-input-group.block.scss

```diff
@@ -0,0 +1,48 @@
+.fhi-input-group {
+    margin-bottom: $fhi-space-3;
+}
+
+@include media-breakpoint-down(sm) {
+    .fhi-input-group {
+        &__col-1 {
```

Comment:
Since this isn't a generic gridsystem I think a more spesific class name is better, eg. &__col-label, &__col-form-control